test/NuGet.Clients.Tests/NuGet.CommandLine.Test/Util.cs

```diff
@@ -1203,5 +1203,29 @@ public static void TestCommandInvalidArguments(string command)
             // Verify traits of help message in stdout
             Assert.Contains("usage:", result.Item2);
         }
+
+        //This is a temperary change since the VS on CI doesn't have the latest hostfxr, it could not resolve the preview SDK.
+        //Will be removed when CI has the latest version of VS, or SDK3.0 released RTM version.
+        //Create a global.json file in temperary testing folder, to make sure msbuild could choose preview version of SDK.
+        public static void CreateTempGlobalJson(string solutionRoot)
```

Comment:
Make sure you have a tracking issue.

Reply:
Thanks! Added a tracking issue https://github.com/NuGet/Home/issues/8387